latest/docs-ref-conceptual/install-azure-cli.md

```diff
@@ -26,28 +26,28 @@ For information about the latest release, see the [release notes](release-notes-
 
 ## macOS
 
-> [!WARNING]
-> The Homebrew formula for the Azure CLI, `azure-cli`, is currently out of date and will install a previous version.
+1. Install `curl` via [homebrew](https://brew.sh/).
+
+   ```bash
+   brew install curl
```

Comment:
I didn't need to install curl via brew for macOS. It was just there. I don;t think we need this step. Can we verify?